concordium-consensus/src/Concordium/External.hs

```diff
@@ -527,6 +535,7 @@ toReceiveResult ResultContinueCatchUp = 10
 toReceiveResult ResultEarlyBlock = 11
 toReceiveResult ResultMissingImportFile = 12
 toReceiveResult ResultConsensusShutDown = 13
+toReceiveResult ResultExpiryTooLate = 14
```

Comment:
This datatype must be kept in sync with (1) the comment above, and (2) the type ConsensusFfiResponse in concordium-node/src/consensus_ffi/helpers.rs